server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java

```diff
@@ -173,6 +172,9 @@ public final class ZooKeeperCommandExecutor
 
     private MetadataService metadataService;
 
+    // Failing to acquire a lock is a critical problem, so we wait as much as we can.
+    private long lockTimeoutMillis = TimeUnit.MINUTES.toMillis(1);
```

Comment:
Q) Would it be better to increase the default timeout for the server and client?

Reply:
If 'the server and client' means Central Dogma API server and client, then I guess not. Failing to acquire a lock in time is a critical error that shouldn't occur frequently and increasing API timeout because this situation is an overkill in my opinion.